src/app/components/client/ComboBox.module.scss

```
@@ -0,0 +1,55 @@
+@import "../../tokens";
+
+.comboBox {
+  background: $color-white;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+
+  .inputField {
+    border: 1px solid $color-grey-30;
+    border-radius: $border-radius-sm;
+    color: $color-black;
+    padding: calc($spacing-sm * 1.5) $spacing-md;
+
+    &::placeholder,
+    &.noValue {
+      color: $color-grey-40;
+    }
+
+    &.hasError {
+      border-color: $color-error;
+    }
+  }
+
+  .inputLabel {
+    font-weight: 600;
+    margin-bottom: $spacing-sm;
+  }
+
+  .inputMessage {
+    background: inherit;
+    bottom: 0;
+    color: $color-error;
+    font: $text-body-xs;
```

suggestion: Use `font-size` here, as you're only setting the font size. 

```suggestion
    font-size: $text-body-xs;
```

---

The token has the entire font styles `400 clamp(10px, 0.975svw, 12px) / 1.5 var(--font-inter),
  sans-serif;` and assigning it to `font-size`. Since we don’t have only font sizes added to `tokens.scss` I’m hesitant to add them with this PR.

---

OH! My bad. You have it right. 😓